lib/ui/semantics/semantics_node.h

```diff
@@ -86,6 +86,7 @@ struct SemanticsNode {
   int32_t actions = 0;
   int32_t textSelectionBase = -1;
   int32_t textSelectionExtent = -1;
+  int32_t platformViewId = -1;
```

Comment:
@amirh Can I use `-1` to represent and invalid id?

Reply:
Yes, the IDs starts at 0 here https://github.com/flutter/flutter/blob/01a29b85eff7dd4b2d4958f61adf1840042471cc/packages/flutter/lib/src/services/platform_views.dart#L28

We should probably add a comment on that line in the framework mentioning -1 is reserved.

Reply:
I’ll add that comment in the framework PR.